application.py

```diff
@@ -30,7 +36,20 @@ def index():
 def register():
     if request.method == "POST":
         username = request.form['username']
+        email = request.form['email']
         password = request.form['pswd']
-        print(username, password)
-        return render_template("index.html",data=username)
+        registerTime = datetime.now()
+        print(username,email,registerTime,password)
+
+        queryResultSet = Users.query.filter_by(email=email).all()
+
+        responseMessage = ""
+        if(len(queryResultSet) > 0):
+            responseMessage = "The email id you have used already exists!"
+        else:
```

Comment:
Simplyfy this by catching an exception at the time of inserting the record. SQLAlchemy throws the IntegrityError.